src/wp-includes/customize/class-wp-customize-nav-menus-panel.php

```diff
@@ -98,4 +98,27 @@ protected function content_template() {
 		<li class="customize-control-title customize-section-title-nav_menus-heading"><?php _e( 'Menus' ); ?></li>
 		<?php
 	}
+
+	/**
+	 * Checks required user capabilities and whether the theme has the
+	 * feature support required by the panel.
+	 *
+	 * @since 5.9.0
+	 *
+	 * @return bool False if theme doesn't support the panel or the user doesn't have the capability.
+	 */
+	public function check_capabilities() {
```

Comment:
Another way to do this would have been to implement an `active_callback` which would return the result of `current_theme_supports( 'menus' ) || current_theme_supports( 'widgets' )`. The constructor could have:

```php
$this->active_callback = function () {
    return current_theme_supports( 'menus' ) || current_theme_supports( 'widgets' );
};
```

The end result would be largely the same, however.

Reply:
Turns out that the way this was implemented doesn't work as intended. A PHP notice is triggered. See https://core.trac.wordpress.org/ticket/54905#comment:36